src/autoschedulers/adams2019/Makefile

```diff
@@ -82,7 +82,7 @@ $(BIN)/libautoschedule_adams2019.$(SHARED_EXT): $(SRC)/AutoSchedule.cpp \
 	@mkdir -p $(@D)
 	$(CXX) -shared $(USE_EXPORT_DYNAMIC) -fPIC -fvisibility=hidden -fvisibility-inlines-hidden $(CXXFLAGS) $(OPTIMIZE) -I $(BIN)/cost_model $(filter-out %.h $(LIBHALIDE_LDFLAGS),$^) -o $@ $(HALIDE_SYSTEM_LIBS) $(HALIDE_RPATH_FOR_LIB)
 
-$(BIN)/retrain_cost_model: $(SRC)/retrain_cost_model.cpp \
+$(BIN)/halide_retrain_cost_model: $(SRC)/halide_retrain_cost_model.cpp \
```

Comment:
These are internal utilities, not really things that should be installed in any non-Halide-prefixed bin dir

Reply:
Yep, i guessed as much, which is why i'm //trying// to do all this,
because as it stands right now, now na[t]ive `ninja install` puts them into `${CMAKE_INSTALL_PREFIX}/bin`